docs/technical/editor-integration.md

```diff
@@ -251,29 +258,32 @@ These editors are currently supported:
  - [Atom](https://atom.io/)
  - [MacVim](https://macvim-dev.github.io/macvim/)
  - [Visual Studio Code](https://code.visualstudio.com/) - both stable and Insiders channel
+ - [Visual Studio Codium](https://vscodium.com/)
  - [Sublime Text](https://www.sublimetext.com/)
  - [BBEdit](http://www.barebones.com/products/bbedit/)
  - [PhpStorm](https://www.jetbrains.com/phpstorm/)
  - [RubyMine](https://www.jetbrains.com/rubymine/)
- - [IntelliJ IDEA](https://www.jetbrains.com/idea/)
  - [TextMate](https://macromates.com)
  - [Brackets](http://brackets.io/)
      - To use Brackets the Command Line shortcut must be installed.
        - This can be done by opening Brackets, choosing File > Install Command Line Shortcut
  - [WebStorm](https://www.jetbrains.com/webstorm/)
  - [Typora](https://typora.io/)
+ - [CodeRunner](https://coderunnerapp.com/)
  - [SlickEdit](https://www.slickedit.com)
+ - [IntelliJ IDEA](https://www.jetbrains.com/idea/)
+ - [Xcode](https://developer.apple.com/xcode/)
```

Comment:
Nice! Thanks for adding the missing editors to the lists! 😄 

Can you also add Android Studio to the list of mac editors? I just merged https://github.com/desktop/desktop/pull/9332 and forgot to check that...

Reply:
Seems like it was added in #9332, just merged from upstream to resolve this